apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet.IntegrationTests/ErrorReporting/ExceptionLoggerTest.cs

```diff
@@ -42,12 +43,19 @@ public ExceptionLoggerTest()
         }
 
         [Fact]
-        public async Task ErrorReportingTest()
+        public async Task ErrorReportingTest_ErrorReporting()
+            => await ErrorReportingTest(TestServer.Create<ReportToErrorReportingApplication>());
+
+        [Fact]
+        public async Task ErrorReportingTest_Logging()
+            => await ErrorReportingTest(TestServer.Create<ReportToLoggingApplication>());
+
+        public async Task ErrorReportingTest(TestServer testServer)
         {
             DateTime startTime = DateTime.UtcNow;
 
             // Create a test server and make an http.
-            using (TestServer server = TestServer.Create<ErrorReportingApplication>())
+            using (TestServer server = testServer)
```

Comment:
Would it be better to make this method generic and then just use `TestServer.Create<T>()` here?